gui/src/components/Connections.tsx

```diff
@@ -17,7 +17,7 @@ interface Connection {
 
 export function Connections() {
   const { data: connections, mutate } =
-    useInvoke<Connection[]>("get_connections");
+    useInvoke<Connection[]>("peers_get_all");
```

Comment:
It feels like we are using `peer` and `connection` interchangeably. Don't we want some consistency here?

Reply:
true. I renamed some things while working on this, but didn't do it everywhere

do you have a preference? I think Peer is better here, since it conveys better that it's an external client connected to us

Reply:
But it's a client (browser tab) server (our backend) relationship. But either way, you choose.

Reply:
my problem with "Connection" btw, is that we have many types of connections already (e.g.: to an ETH provider).
perhaps "Client"?

Reply:
"Connected Tabs" myabe? or "Connected Websites"?

Reply:
it's neither of those. iframes create their own connections as well

Reply:
🙈

Reply:
renamed to peers (not that I love it, but because we can't keep this thread forever xD)